broker-daemon/orderbook/index.js

```diff
@@ -337,6 +341,39 @@ class Orderbook {
       sequence
     }
   }
+
+  /**
+   * Updates retry count and calculates next delay period if the
+   * MAX_RETRY_INTERVAL hasn't been surpassed
+   *
+   * @private
+   * @return {void}
+   */
+  _updateExponentialDelayPeriod () {
+    if (this._reachedMaxInterval) {
+      return
+    }
+
+    this._retries++
+    const newDelay = (2 ** this._retries - 1) * 1000
+    if (newDelay > MAX_RETRY_INTERVAL) {
+      this._delay = MAX_RETRY_INTERVAL
+      this._reachedMaxInterval = true
+    } else {
+      this._delay = newDelay
+    }
+  }
+
+  /**
+   * Resets exponential backoff parameters to initial state
+   * @private
+   * @return {void}
+   */
+  _resetBackoff () {
```

Comment:
small stylistic note: we're shying away from the `_` prefix as "private" indicator, instead preferring either truly private methods (i.e. not exposed on the object) or just marked as `@private` via jsdoc.

Reply:
Gotcha, since I'm new to Node culture I'm curious if this is a preference of the team or that's idiomatic JavaScript? Googling around seems to have mixed results.

Reply:
Consensus is mixed. We use [StandardJS](https://standardjs.com) which is silent on the topic. We've decided as a team to stay away from the `_` prefix (although there are still a couple of places where it exists in the codebase).